packages/chart/src/components/LineChart/components/LineChart.BumpCircle.tsx

```diff
@@ -0,0 +1,107 @@
+import { useState } from 'react'
+import { Group } from '@visx/group'
+import { type Column } from '@cdc/core/types/Column'
+import React from 'react'
+
+
+const LineChartBumpCircle = props => {
+  const { config, xScale, yScale, parseDate } = props
+  const [tooltipID] = useState<string>('')
```

Comment:
This is never updated so it should be removed.

Reply:
removed tooltipID line